newIDE/app/src/UI/Carousel.js

```diff
@@ -0,0 +1,329 @@
+// @flow
+import * as React from 'react';
+import { Trans } from '@lingui/macro';
+import { makeStyles } from '@material-ui/styles';
+import GridList from '@material-ui/core/GridList';
+import { GridListTile, Paper } from '@material-ui/core';
+import ArrowBackIos from '@material-ui/icons/ArrowBackIos';
+import ArrowForwardIos from '@material-ui/icons/ArrowForwardIos';
+import { Skeleton } from '@material-ui/lab';
+
+import Text from './Text';
+import { Column, Line, Spacer } from './Grid';
+import GDevelopThemeContext from './Theme/ThemeContext';
+import { useResponsiveWindowWidth } from './Reponsive/ResponsiveWindowMeasurer';
+
+type Thumbnail = {|
+  id: string,
+  title: string,
+  description: string,
```

Comment:
is the description needed for a Thumbnail?

Reply:
You're right, I thought I would need it to add a description for accessibility purposes. Maybe I can add a `title` prop to the item title. What do you think?

Reply:
Descriptions can be long, not sure they would have their place as the title attribute of the item. Maybe I'm wrong?
I'd say remove it for the moment

Reply:
Could be great to have a formation about accessibility to better understand that.